x-pack/plugin/ml/qa/native-multi-node-tests/build.gradle

```diff
@@ -15,6 +15,7 @@ dependencies {
   javaRestTestImplementation project(path: xpackModule('monitoring'))
   javaRestTestImplementation project(path: xpackModule('transform'))
   javaRestTestImplementation project(path: xpackModule('rank-rrf'))
+  javaRestTestImplementation project(path: xpackModule('rank-text-similarity'))
```

Comment:
CI was [failing](https://buildkite.com/elastic/elasticsearch-pull-request/builds/23426#01903b1b-a5fa-48f7-acd2-b1b7e0a4fac4) - found a possible resolution similar to the one for `rank-rrf` as per [Slack thread](https://elastic.slack.com/archives/C8UUBNASY/p1710426954194729?thread_ts=1710316157.288789&cid=C8UUBNASY).